packages/rocketchat-ui-flextab/client/tabs/userInfo.js

```diff
@@ -0,0 +1,526 @@
+/* globals RoomRoles, UserRoles*/
+import moment from 'moment';
+import toastr from 'toastr';
+
+Template.userInfo.helpers({
+	name() {
+		const user = Template.instance().user.get();
+		return user.name ? user.name : TAPi18n.__('Unnamed');
+	},
+
+	username() {
+		const user = Template.instance().user.get();
+		return user.username;
+	},
+
+	email() {
+		const user = Template.instance().user.get();
+		if (user.emails && user.emails[0] && user.emails[0].address) {
+			return user.emails[0].address;
+		}
+	},
+
+	utc() {
+		const user = Template.instance().user.get();
+		if (user.utcOffset != null) {
+			if (user.utcOffset > 0) {
+				return `+${user.utcOffset}`;
+			}
+			return user.utcOffset;
+		}
+	},
+
+	lastLogin() {
+		const user = Template.instance().user.get();
+		if (user.lastLogin) {
+			return moment(user.lastLogin).format('LLL');
+		}
+	},
+
+	createdAt() {
+		const user = Template.instance().user.get();
+		if (user.createdAt) {
+			return moment(user.createdAt).format('LLL');
+		}
+	},
+
+	canDirectMessage(username) {
+		return RocketChat.authz.hasAllPermission('create-d') && (Meteor.user.username !== username);
```

Comment:
This will fail: `Meteor.user.username !== username`. Use `Meteor.user() && Meteor.user().username !== username`;